### ISSUE TITLE
NVDA - cannot make svgs that are object src accessible

### ISSUE THREAD

**Issue by hmig** [nvaccess/nvda#6246]
There is no way to access the content or accessibly deal with an SVG that is included on a page as an ```<object>``` src in NVDA. 

In Safari + VO and Narrator + Edge/FF it works by default.  With JAWS, I had to add aria-label and then it would work. But, with NVDA, there is nothing I was able to do to make the SVG get picked up. The only alternative is to a hidden text for all SRs and to hide the ```<object>``` from them. :(

Testing examples can be found https://jsbin.com/lazike.

**Comment by Adriani90** [comment]
I can reproduce this issue in Firefox 65.0.2 and Edge. In Chrome, every svg is recognized properly by NVDA. In IE, only the label and description of the first svg is not recognized by NVDA. So, as usual, this is very inconsistent among browsers.
@jcsteh any idea why NVDA does not recognize the first two SVGs in Firefox at all?

**Comment by Adriani90** [comment]
@jcsteh  This is now working as expected in all three cases in Chrome canary 131 and MS Edge 129. Only Firefox does not work in the first two test cases.

**Comment by Adriani90** [comment]
Might be very related to #6103.
